src/test/returntrip.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+import uuidv4 from 'uuidv4';
+import ReturnTrip from '../controllers/returnTrip';
+//import  { should } from 'chai';
+import Token from './__MOCK__/Tokens';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+// const { Token } = tokens;
+
+
+describe('Testing ReturnTrip Endpoints', () => {
+    it('Return Trip Controller should exist', () => {
+        ReturnTrip.should.exist;
+    });
+
+    it('ReturnTripRequest method (POST) should exist', () => {
+        ReturnTrip.returnTripRequest.should.exist;
+    });
+
+    it('ReturnTripRequest method (POST) should create a Return Trip', (done) => {
+        chai.request(server)
+            .post('/api/v1/trips/request')
+            .set("Authorization", Token)
+            .send({
+                currentOfficeLocation: 'abuja',
+                tripId: uuidv4(),
+                userId: '567',
+                departureDate: '2019-09-12',
+                returnDate: '2019-09-26',
+                accomodation: 'transcop',
+                reason: 'Holiday',
+                tripType: 'Return-trip',
+                requestStatus: 'pending',
+                destination: 'Lagos',
```

Comment:
Destination should be array of integers and current location should be an integer representing the current office location